todoapp-main/src/main/java/ru/todoapp/service/UserService.java

```diff
@@ -44,6 +42,8 @@ public void handle(RegisterRequestDTO registerRequestDTO) {
             userRepository.saveUser(UserEntity.of(registerRequestDTO));
             sendRequestResultDTO(registerRequestDTO.getRequestUUID(), RequestStatus.SUCCESS, null);
         }
+
+        requestRepository.save(RequestEntity.of(registerRequestDTO));
```

Comment:
Why you put this at the end of the method? I understand that it's changed nothing, but is not it more logical to save request right it has been received?

Reply:
Yes and no, If something fails completely Kafka with its current config will continue sending me request that will fail, the main body of the request will not be saved to its own table (user/task) but it will fuck up our request table untill it will timeout + @aeDeaf said that that table and everything with it will go at the end anyway